Neos.Flow/Documentation/TheDefinitiveGuide/PartIII/ResourceManagement.rst

```diff
@@ -449,6 +449,27 @@ this. To use this feature, just pass the resource hash:
 You are encouraged to use this stream wrapper wherever you need to access a static or
 persistent resource in your PHP code.
 
+Package Stream Wrapper
+======================
+
+Packages often rely on files from other packages and in some cases those might not be resources.
+Typical use cases are NodeTypes, Tests, YAML or other data files.
+Similar to the resources it not advisible to build pathes using the ``FLOW_PATH_*`` constants or
+include files via relative pathes from your package. The package stream wrapper offers a more
+convenient and safe way to achieve this.
+
+The following example reads the content of the file
+``Acme.Demo/NodeTypes/Document.yaml`` into a variable:
```

Comment:
To not confuse anyone, that should refer to `Acme.Demo/Configuration/NodeTypes/Document.yaml`, I'd say.

Reply:
Isnt `NodeTypes` a new top level folder?

Reply:
Would be news to me. 🗞

Reply:
Yes, that's correct since 7.2: https://www.neos.io/blog/neos-flow-72-released.html#NodeTypes-in-Subfolders

Reply:
hmm i was wrong - not in flow context ...

flow doesnt care about NodeTypes

Reply:
@mhsdesign no, you were correct and so is the snippet

Reply:
yes but this is a flow documentation - and flow is not about neos...

Reply:
> yes but this is a flow documentation

Ah right, good point :)

> I think that's ugly AF.

Not related to this change, but for the record: There are reasons for it. One of it being that it is not a (context specific) configuration and that we wanted to make NodeType and Integration Fusion co-locatable (i.e. put it in the same folder and don't require some `NodeType.*` prefix). See the full discussion at https://github.com/neos/neos-development-collection/issues/3000

Reply:
Actually even if flow is not about Neos it is the most common use-case. That is why i used this as example.